src/auto_tune/evo/individual.py

```diff
@@ -0,0 +1,30 @@
+import numpy as np
+
+from sklearn.model_selection import cross_val_score
+
+
+class Individual(object):
+    def __init__(self, model, genome, data, target, phenome=None):
+        self.model = model
+        self.genome = genome
+        self.phenome = phenome
+        self.data = data
+        self.target = target
+
+        if self.phenome is None:
+            self.phenome = [g.random() for g in self.genome]
+
+        for i in range(len(self.phenome)):
+            setattr(self.model, self.genome[i].param, self.phenome[i])
+
+        self.fit()
+
+    def fit(self):
```

Comment:
is this method intended for only be used in the `__init__`? if yes ... perhaps it will be better to rename it to `_fit`.